journal/updated/09-chapter-8.adoc

```diff
@@ -299,58 +304,58 @@ Several strangers were there, and I had satisfaction in being with them,
 being favoured with a sense of Divine life, whereby I had access to the throne of grace,
 and witnessed a renewal of strength in the inner man,
 although I feel a great decay of bodily strength,
-and was ill part of the time of being there.
+and was ill part of the time I was there.
 
-In the fourth month, 1792, was again at our Quarterly Meeting,
+In the fourth month, 1792, I again was at our Quarterly Meeting,
 and in the several sittings thereof we were favoured
 with a sense of the love of our Lord,
-who still is with those that are gathered together in his name.
+who still is with those that are gathered together in His name.
 
 Fourth month 29th, 1793.
 Attended our Quarterly Meeting at Nottingham,
-and 1st of fifth month our Monthly Meeting at Loughborough;
-this little journey I was enabled to perform beyond expectation,
+and 1st of fifth month our Monthly Meeting at Loughborough.
+This little journey I was enabled to perform beyond expectation,
 and felt my mind under the covering of the Spirit of the Lord, and returned in peace.
-My complaints too not feeling so grievous as in months past,
+My complaints not feeling so grievous as in months past,
```

Comment:
maybe include the word "physical" so that it's clear: My physical complaints not feeling so grievous....